e3db/types/note_options.py

```diff
@@ -87,6 +87,42 @@ def to_json(self) -> dict:
 
         return to_serialize
 
+    @staticmethod
+    def decode(json):
+        """
+        Decodes a JSON response into a NoteOptions object. 
+
+        Parameters
+        ----------
+        json : dict
+            JSON data to be decoded into a NoteOptions object
+
+        Returns
+        -------
+        note : e3db.NoteOptions
+            A NoteOptions object that contains the contents from json 
+        """
+        client_id = json['client_id'] if 'client_id' in json else None
+        max_views = json['max_views'] if 'max_views' in json else None
+        expiration = json['expiration'] if 'expiration' in json else None
+        expires = json['expires'] if 'expires' in json else None
```

Comment:
There may be some value in having this None be a False. Functionally I think they are similar but for end users it may be more clear if they are inspecting fields